.github/workflows/identity-regression-test.yml

```diff
@@ -96,6 +134,27 @@ jobs:
           ./bin/camunda &
           sleep 60
 
+      - name: Start Camunda (OIDC config for mapping-rules)
+        if: matrix.case == 'mapping-rules.http'
+        run: |
+          cd ./dist/target/camunda-zeebe-8.8.0-SNAPSHOT
+          export ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_CLASSNAME=io.camunda.exporter.CamundaExporter && \
+          export ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_ARGS_CONNECT_URL=http://localhost:9200  && \
+          export ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_ARGS_BULK_SIZE=100  && \
+          export ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_ARGS_INDEX_SHOULDWAITFORIMPORTERS=false  && \
+          export CAMUNDA_SECURITY_AUTHENTICATION_METHOD=OIDC  && \
+          export CAMUNDA_SECURITY_AUTHENTICATION_OIDC_CLIENTID=${{ steps.secrets.outputs.CLIENT_ID}}  && \
+          export CAMUNDA_SECURITY_AUTHENTICATION_OIDC_CLIENTSECRET=${{ steps.secrets.outputs.CLIENT_SECRET}}  && \
+          export CAMUNDA_SECURITY_AUTHENTICATION_OIDC_ISSUERURI=${{ steps.secrets.outputs.ISSUER_URI}}  && \
```

Comment:
❓ This is running against our entra ID instance, should we have a less volatile tenant/dedicated client for these tests?

Reply:
Works for me :) just something to bear in mind in the future if this workflow breaks!